### ISSUE TITLE
The clean command is broken

### ISSUE THREAD

**Issue by dharmavagabond** [phpbrew/phpbrew#561]
Class [PhpBrew\Command\CleanCommand](https://github.com/phpbrew/phpbrew/blob/4db68e8038620643629e45d29205ad13ec5f0b96/src/PhpBrew/Command/CleanCommand.php#L7) misses `use PhpBrew\Utils`. When I added the import, it threw the following error:

```sh
PHP Fatal error:  Interface 'Universal\ClassLoader\ClassLoader' not found in phar:///Users/Dharma/Desktop/phpbrew/phpbrew/Universal/ClassLoader/SplClassLoader.php on line 2
PHP Stack trace:
PHP   1. {main}() /Users/Dharma/Desktop/phpbrew/phpbrew:0
PHP   2. require() /Users/Dharma/Desktop/phpbrew/phpbrew:3

Fatal error: Interface 'Universal\ClassLoader\ClassLoader' not found in phar:///Users/Dharma/Desktop/phpbrew/phpbrew/Universal/ClassLoader/SplClassLoader.php on line 2

Call Stack:
    0.0002     236080   1. {main}() /Users/Dharma/Desktop/phpbrew/phpbrew:0
    0.0020     408536   2. require('phar:///Users/Dharma/Desktop/phpbrew/phpbrew/Universal/ClassLoader/SplClassLoader.php') /Users/Dharma/Desktop/phpbrew/phpbrew:3
```

**Comment by jhdxr** [comment]
I tried `phpbrew clean` as the documents said, and get the following error:
> Insufficient arguments for command 'clean', which requires 1 arguments, 0 given.

and `phpbrew clean php-5.6.12`
> Makefile not found in path 

I think we should keep the README updated or many people will get confused.

**Comment by jhdxr** [comment]
```
[root@localhost phpbrew]# phpbrew clean php-5.6.12
#0  PhpBrew\Tasks\MakeTask->make(, clean) called at [/root/proj/phpbrew/src/PhpBrew/Tasks/MakeTask.php:26]
#1  PhpBrew\Tasks\MakeTask->clean(PhpBrew\Build Object ([name] => php-5.6.12,[version] => php-5.6.12,[sourceDirectory] => ,[installPrefix] => ,[*configDirectory] => ,[phpEnvironment] => 1,[settings] => PhpBrew\BuildSettings\BuildSettings Object ([variants] => Array (),[disabledVariants] => Array (),[extraOptions] => Array ()),[state] => )) called at [/root/proj/phpbrew/src/PhpBrew/Command/CleanCommand.php:47]
#2  PhpBrew\Command\CleanCommand->execute(php-5.6.12)
```

and there is a comment in Build.php
```
// TODO: find the install prefix automatically
```

It seems the clean command uses unfinished feature

**Comment by jhdxr** [comment]
@dharmavagabond if you are tryint to use a version cloned from git, you need to run `composer install` to make sure you have all the deps ready.

**Comment by dharmavagabond** [comment]
@jhdxr Sorry for the late reply. I tried both ways, by downloading the phar, which thew the errors you mentioned, and cloning the repo. With the latter, after installing the the composer dependencies (as you also mentioned) and making the phar, it threw ```PHP Fatal error:  Interface 'Universal\ClassLoader\ClassLoader' not found```.

**Comment by marcioAlmada** [comment]
Hi, this is caused by a BCB on some underlying package and will be fixed ASAP the PR above gets merged :)

**Comment by c9s** [comment]
Fixed and released in 1.20.2
